pynav/lib/models.py

```diff
@@ -83,8 +83,44 @@ def covariance(
         return L @ self._Q @ L.T
 
 
+class Imu:
```

Comment:
Should there be anything forcing the user to implement a plus and a copy operator, if they implement their own input types?

Reply:
we should have to introduce an abstract class for this i think. at the moment, if these arnt implemented it will fail pretty loudly, but this is definitely something to keep in mind